data/xml/C94.xml

```diff
@@ -1115,23 +1115,23 @@
       <url>C94-2178</url>
     </paper>
     <paper id="179">
-      <title>A FORMAL REPRESENTATION OF THE THEMATIC-RHEMATIC STRUCTURE OF SENTENCES BASED ON A TYPED A-CALCULUS</title>
+      <title>A Formal Representation of the Thematic-Rhematic Structure of Sentences Based on a Typed Λ-Calculus</title>
```

Comment:
It is written with a capital lambda in the paper; I changed it to lowercase lambda thinking that BibTeX would not know how to lowercase it correctly. But then title_case.py uppercased it again. There is another paper where the same thing must have happened. Any opinions @mjpost @aryamccarthy ?

Reply:
I've only ever seen lambda lowercased in this setting. I suggest leaving it lowercased and case-unprotected.